pages/dashboard/views/index.jsx

```diff
@@ -30,38 +22,39 @@ const Invitation = ({ token, establishment }) => (
 const Index = ({
   tabs,
   progress,
-  profile: {
-    firstName,
-    establishments,
-    invitations
-  }
+  profile
 }) => {
+
   return <Fragment>
     <Header
-      title={<Snippet name={firstName}>pages.dashboard.greeting</Snippet>}
+      title={<Snippet name={profile.firstName}>pages.dashboard.greeting</Snippet>}
     />
-    <div className="govuk-grid-row">
-      <div className="govuk-grid-column-full">
-        <h2><Snippet>pages.dashboard.tasks</Snippet></h2>
-        <TaskList tabs={ tabs } progress={ progress } />
 
-        {
-          !!establishments.length && <Fragment>
-            <h2>Establishments</h2>
+    <h3><Snippet>pages.dashboard.tasks</Snippet></h3>
+    <TaskList tabs={ tabs } progress={ progress } />
+    {
+      !!profile.establishments.length && <Fragment>
+        <h3>Establishments</h3>
+        <PanelList panels={sortBy(profile.establishments, 'name').map(establishment => {
+          return (
+            <div className="landing" key={establishment.id}>
```

Comment:
rather than wrapping this in a div _just for styling_ - cant you target the expanding panels in .panel-list? `.panel-list 
 > .expanding-panel {  styles }`

Reply:
done